pkg/kubelet/metrics/metrics.go

```diff
@@ -50,6 +50,13 @@ var (
 			Help:      "Latency in microseconds to sync all pods.",
 		},
 	)
+	StartPodWorkerLatency = prometheus.NewSummary(
+		prometheus.SummaryOpts{
+			Subsystem: kubeletSubsystem,
+			Name:      "start_pod_worker_latency_microseconds",
+			Help:      "Latency in microseconds from seeing a pod to starting a worker.",
+		},
```

Comment:
The reason an aggregate isn't very helpful is because we don't care as much about the starting 30 pods case as we do about the starting 1 pod after 30 case. The former is known to be slow for various reasons (probably not in this pipeline though).